plugins/discourse-assign/spec/system/assign_topic_spec.rb

```diff
@@ -3,19 +3,18 @@
 describe "Assign | Assigning topics", type: :system do
   let(:topic_page) { PageObjects::Pages::Topic.new }
   let(:assign_modal) { PageObjects::Modals::Assign.new }
-  fab!(:staff_user) { Fabricate(:user, groups: [Group[:staff]]) }
-  fab!(:admin)
+  fab!(:admin1) { Fabricate(:admin) }
```

Comment:
We have various rules for showing small posts, using admin user here rules out mixed configurations.